### PR TITLE
Merge v7 staging into v7 prod

### PR DESCRIPTION
**Pull request by luisccastillo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ Worker pools are made up of multiple individual machines. They are led by a Pool
 
 **The Pool Manager freely determines the pricing to execute computational task for its workers.**
 
-Pool manager, while not doing the actual computation, receive a fee for the management of the worker pool. Pool manager compete to attract workers to their worker pool by providing efficient management and guaranteeing earnings for workers.
+Pool manager, while not doing the actual computation, receives a fee for the management of the worker pool. Pool managers compete to attract workers to their worker pool by providing efficient management and guaranteeing earnings for workers.
```
Comment (mcornaton) [inline README.md:54]: Why having "pool manager" the first time and "pool managers" the second time? I"d prefer both to be plural :)

Comment (luisccastillo) [reply README.md:54]: @SeddikBellamine

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Review by zguesmi — APPROVED** [review]

**Comment by mcornaton** [comment]
This is not up-to-date: Scone image version should be 5.3.11, not 5.3.9.

**Comment by zguesmi** [comment]
I think the version `5.3.9` is compatible with v7, latest versions are more fresh than v7.

**Pull request merged**